tests/components/calendar/test_init.py

```diff
@@ -419,35 +424,25 @@ async def test_list_events_service(
         blocking=True,
         return_response=True,
     )
-    assert response == {
-        "events": [
-            {
-                "start": "2023-06-22T05:00:00-06:00",
-                "end": "2023-06-22T06:00:00-06:00",
-                "summary": "Future Event",
-                "description": "Future Description",
-                "location": "Future Location",
-            }
-        ]
-    }
+    assert response == snapshot
```

Comment:
I think the purpose of this test was to compare that the response is equal for every call we do. We are technically still checking that, but I can imagine that we forget that this was the test and just accept it. (For example some code can now change that the snapshots would be A-A-B, while they should be all A-A-A. If there is a legit reason to make the end result B-B-B, they should also take in account that the other cases should be fixed as well).

But I do love tests without data structures, so let me have a look to see if we can create 1 data structure for the whole test.